src/core/datastructures/image/layerram.cpp

```diff
@@ -42,7 +42,7 @@ LayerRAM::LayerRAM(size2_t dimensions, LayerType type, const DataFormatBase* for
 bool LayerRAM::copyRepresentationsTo(LayerRepresentation* targetLayerRam) const {
     static DataWriterType<Layer>* layerWriter_ = InviwoApplication::getPtr()
                                                      ->getDataWriterFactory()
-                                                     ->getWriterForTypeAndExtension<Layer>("png")
+                                                     ->getWriterForTypeAndExtension<Layer>("bmp")
```

Comment:
This is a horrible hack here... might be nice to add a comment explaining what is going on

Reply:
The whole thing with using a DataWriterType<Layer> for resizing/copying is very hacky, but I will add a comment.